.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,43 @@
+### 🧪 Pruebas
+
+#### Manual Testing
+- [ ] 
+- [ ] 
+- [ ] 
+
+#### Compilación
+- [ ] `./mvnw clean compile` - SUCCESS
+- [ ] Aplicación inicia sin problemas
+
+###  Documentación
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)